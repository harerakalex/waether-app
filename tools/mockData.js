const weathers = {
  temp: 23,
  pressure: 30,
  humidity: 30
};

// Using CommonJS style export so we can consume via Node (without using Babel-node)
module.exports = {
  weathers
};
