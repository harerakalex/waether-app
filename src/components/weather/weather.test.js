import React from 'react';
import WeatherItem from './weatherItem';
import renderer from 'react-test-renderer';
import { weathers } from '../../../tools/mockData';

it('should pass', () => {
  const tree = renderer.create(<WeatherItem todo={weathers} />);
  expect(tree).toMatchSnapshot();
});
