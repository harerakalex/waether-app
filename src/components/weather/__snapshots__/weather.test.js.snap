// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should pass 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Current weather in your location
  </h1>
  <p>
    <small>
      Temp: 
    </small>
    <strong>
      23
    </strong>
  </p>
  <p>
    <small>
      pressure: 
    </small>
    <strong>
      30
    </strong>
  </p>
  <p>
    <small>
      humidity: 
    </small>
    <strong>
      30
    </strong>
  </p>
</div>
`;
